src/beast/test/asio/streambuf.cpp

```diff
@@ -0,0 +1,296 @@
+//
```

Comment:
It doesn't look like `streambuf.cpp` is compiled with rippled (I am unable to run the `streambuf` test and when I purposely made a syntax error in this file scons happily compiled rippled).